stacks-signer/src/v0/signer.rs

```diff
@@ -251,7 +251,7 @@ impl Signer {
         block_proposal: &BlockProposal,
         miner_pubkey: &Secp256k1PublicKey,
     ) {
-        debug!("{self}: Received a block proposal: {block_proposal:?}");
+        info!("{self}: Received a block proposal: {block_proposal:?}");
```

Comment:
I guess this one can remain a debug since the next log gives the same info and more.

Reply:
They are both still info, right?

Reply:
Let's make this one `debug!`.

Reply:
My bad, I thought that it had a suggestion on it and I wanted to confirm the change in the code. But I only clicked resolve comment. I've pushed it now.